hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAManagerImpl.java

```diff
@@ -337,8 +337,8 @@ void reloadSCMState()
   public void shutdown() throws IOException {
     if (ratisServer != null) {
       ratisServer.stop();
-      ratisServer.getSCMStateMachine().close();
       grpcServer.stop();
+      ratisServer.getSCMStateMachine().close();
```

Comment:
Can you please explain this change in order of statements?

Reply:
if we want stateMachine#close to terminate the running process, i think it is better to stop grpc server before the process is terminated